HLTrigger/Egamma/plugins/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<library   file="*.cc" name="HLTriggerEgammaPlugins">
```

Comment:
HLTrigger/Egamma already defines only plugins (in the src directory), you should not use a new 'plugins' directory.